participating.md

```diff
@@ -1,5 +1,70 @@
+- 🎓 Best Undergraduate
+Specific prize for Undergraduate Students, drawn from across the main categories.
+
+
+#### **Prize selection**: To select the prize category that best suits your idea, you should consider the following questions:
+- **What is the main goal of your idea?**: 
+- **What is the main strength of your idea?**:
+- **What is the main challenge of your idea?**: Is it the relevance, the feasibility, the scalability, or the replicability of your idea?
+- **What is the main audience of your idea?**: Is it software practitioners, software users, software stakeholders, or the general public?
+ 
+
+## 💚 I need help on my project
+- **Project support**: Carbon Hack provides various forms of support for your project, such as¹:
+    - **Access to experts**: Our Impact Framework team are here to help! You can request support through the Carbon Hack Discussions forum.
+    - **Resources**: You can access resources such as tutorials, guides, webinars, and case studies on how to use IF and other green software tools and practices. You can find these resources on the IF website.
+    - **Community & Orihect Feedback**: You can connect with other participants, experts, and the organizers of Carbon Hack 24 through the Carbon Hack discussions forum on GitHub, and the social media channels of the Green Software Foundation. You can also join the events and activities organized by Carbon Hack 24, such as ideation sessions, workshops, and networking sessions.
+
+
+## 💚 Registering & Submitting Ideas
+- **Registration process**: To register for Carbon Hack 24, you need to complete the following steps¹:
+    - **Create a team**: You can create a team of up to 5 members, or join an existing team, or participate as an individual. You can use the Carbon Hack 24 site and GitHub repo to find and connect with potential teammates.
+    - **Submit a project idea**: You can to submit a project idea that aligns with the theme and criteria of Carbon Hack 24. You can use the Carbon Hack 24 platform to submit your project idea and get feedback from other participants, mentors, and organizers. Please note, 
+    - **Register your team**: You need to register your team and your project idea on the Carbon Hack 24 website. You need to provide the following information: team name, team members, project name, project description, project category, and contact details.
+ 
+## 💚 What happens after I register, how will you help me?
+- **Post-registration support**: After you register for Carbon Hack 24, you will receive the following support¹:
+    - **Confirmation email**: You will receive a confirmation email with your team and project details, and a link to access the Carbon Hack 24 repo and resources.
+    - **Welcome pack**: You will receive a welcome pack with information and guidance on how to participate in Carbon Hack 24, such as the timeline, the rules, the judging criteria, and the tips and tricks.
```

Comment:
Whatever information you were thinking of putting in this pack, goes in this repo.